pkgs/applications/audio/iconnconfig/default.nix

```
@@ -0,0 +1,46 @@
+{ lib
+, mkDerivation
+, fetchgit
+, qmake
+, alsaLib
+, libusb1
+}:
+
+mkDerivation rec {
+  pname = "IConnConfig";
```

```suggestion
  pname = "iconnconfig";
```

the attr and pname should ideally match.